payments-core/src/test/java/com/stripe/android/view/CardFormViewTest.kt

```diff
@@ -273,11 +350,54 @@ class CardFormViewTest {
             // 1 horizontal divider added below tl_cvc, now it has 2 child views
             assertThat(it.cardMultilineWidget.cvcInputLayout.childCount).isEqualTo(2)
 
+            // divider with width=match_parent is invisible
+            assertThat(it.countryPostalDivider.isVisible).isFalse()
+
             // 1 horizontal divider added below countryLayout, now it has 2 child views
             assertThat(it.countryLayout.childCount).isEqualTo(2)
+        }
+    }
+
+    @Test
+    fun verifyBorderlessNoPostalStyle() {
+        idleLooper()
+        borderlessNoPostalBinding?.let {
+            // no child views added to card_multiline_widget.xml, it still has 2 child views
+            assertThat(it.cardMultilineWidget.childCount).isEqualTo(2)
+            // tl_card_number
+            assertThat(it.cardMultilineWidget.getChildAt(0)).isInstanceOf(
+                CardNumberTextInputLayout::class.java
+            )
+            // second_row_layout
+            assertThat(it.cardMultilineWidget.getChildAt(1)).isInstanceOf(
+                LinearLayout::class.java
+            )
+
+            // 1 horizontal divider added to tl_card_number, now it has 2 child views
+            assertThat(it.cardMultilineWidget.cardNumberTextInputLayout.childCount).isEqualTo(
```

Comment:
I think in other places we have put .isEqualTo on a new line so:
```
assertThat(it.cardMultilineWidget.cardNumberTextInputLayout.childCount)
     .isEqualTo(2)
```

but that is just a personal preference.

Reply:
trying to stay with existing styles in other tests, so will keep them as is